sdk/cmake/msvc.cmake

```diff
@@ -536,7 +538,7 @@ function(add_linker_script _target _linker_script_file)
         add_custom_command(
             # OUTPUT ${_generated_file}
             TARGET ${_target} PRE_LINK # PRE_BUILD
-            COMMAND ${CMAKE_C_COMPILER} /nologo ${_no_std_includes_flag} /D__LINKER__ /EP /c "${_file_full_path}" > "${_generated_file}"
+            COMMAND cl /nologo ${_no_std_includes_flag} /D__LINKER__ /EP /c "${_file_full_path}" > "${_generated_file}"
```

Comment:
Why is it required to change the name of the C compiler? Shouldn't this be already correctly configured in `CMAKE_C_COMPILER` for MSVC builds?

Reply:
So ??

Reply:
Done